tests/main/views/test_direct_award.py

```diff
@@ -200,6 +201,13 @@ def _task_has_link(self, tasklist, task, link):
 
         return len(anchors) == 1
 
+    def _task_has_button(self, tasklist, task, name, value):
+        """Task here refers to the tasklist item number rather than the zero-indexed python array. This feels easier to
+        understand and amend in the context of the tasklist."""
+        anchors = tasklist[task - 1].xpath('.//button[@name="{}" and @value="{}"]'.format(name, value))
```

Comment:
FWIW there's a trick you can do with lxml's `xpath()` here so that you don't have to worry about escaping:

```
.xpath(".//button[@name=$name and @value=$val]", name=name, val=value)
```

Reply:
not worrying about escaping?? 👿 blasphemy 👿

Reply:
Well, y'know, in that it's no longer an issue - with `.format()` there's nothing stopping the inserted string containing `"` or some non-xpath-valid character...